android/app/src/main/java/com/rnpointsdkminimalintegration/MainApplication.java

```diff
@@ -46,7 +37,6 @@ public boolean getUseDeveloperSupport() {
           new MainReactPackage(),
             new GeolocationPackage(),
             new ReactNativePushNotificationPackage(),
-            new RNPermissionsPackage(),
```

Comment:
After building while running the android app, its reporting error as: `ERROR  Error: react-native-permissions: NativeModule.RNPermissions is null. `